### PR TITLE
add a download button

### PR DESCRIPTION
**Pull request by kassiwu**
PLOT -- add a download button

### CODE REVIEW

**Comment by tsangjustin** [comment]
@kassiwu can you please add screen shots of it?

**Comment by kassiwu** [comment]
1. Click download button
screen shot 2018-09-29 at 11 35 21 pm
2. Downloading
screen shot 2018-09-29 at 11 35 51 pm
3. View the downloaded picture
screen shot 2018-09-29 at 11 36 11 pm

File: interface/components/graph/index.coffee [hunk 1]
```diff
@@ -66,4 +66,68 @@ ko.components.register "tf-graph",
 
       element.appendChild chart.element
 
+      download_chart = () ->
+        svg_element = chart.element.querySelector "svg"
```
Comment (tsangjustin) [inline interface/components/graph/index.coffee]: What if there are no charts? Will it still show the download button?

File: interface/components/graph/index.coffee [hunk 2]
```diff
@@ -66,4 +66,68 @@ ko.components.register "tf-graph",
...
+        original_height = svg_element.getAttribute "height"
+        original_width = svg_element.getAttribute "width"
+        svg_element.removeAttribute "height"
```
Comment (tsangjustin) [inline interface/components/graph/index.coffee]: Can you add comment why you removed these attributes?

**Comment by tsangjustin** [comment]
@kassiwu should we close this PR? Haven't made any pushes over a month

**Pull request closed without merging**